pkg/validating-webhooks/agentinstall/v1beta1/infraenv_admission_hook_test.go

```diff
@@ -0,0 +1,219 @@
+package v1beta1
```

Comment:
Also add test if the Spec.ClusterRef is nil in the old and try to change to not nil, and the other way around.